tendrl/alerting/handlers/cluster/cluster_utilization_handler.py

```diff
@@ -0,0 +1,28 @@
+from tendrl.alerting.handlers import AlertHandler
+from tendrl.alerting.objects.cluster_alert import ClusterAlert
+
+
+class ClusterUtilizationHandler(AlertHandler):
```

Comment:
Currently its the same but while resolving https://github.com/Tendrl/alerting/issues/32 this will be different i.e, each of them will have their own message formatting method...

Also, these plugins decide where the associated alert goes in etcd apart from 1 common path i.e, /alerting/alerts . In case of cluster alerts they go to /alerting/clusters/<cluster-id>/... and in case of node related alerts they go to /alerting/nodes/<node-id>/... And going forward if we feel there needs to be custom handling for each/some alert/s, they can be done by overriding the handle method....